helm/object-storage-operator/templates/rbac.yaml

```diff
@@ -45,48 +31,40 @@ rules:
       - events
     verbs:
       - create
-  {{ if eq .Values.managementCluster.provider.kind "capz" -}}
+
+  {{ if eq .Values.managementCluster.provider.kind "capa" -}}
   - apiGroups:
-    - infrastructure.cluster.x-k8s.io
-    resources:
-    - azureclusteridentities
-    - azureclusteridentities/status
-    verbs:
-    - get
-    - list
-    - patch
-    - update
-    - watch
-  - apiGroups:
-    - aadpodidentity.k8s.io
+      - infrastructure.cluster.x-k8s.io
     resources:
-    - azureidentities
-    - azureidentities/status
+      - awsclusters
+      - awsclusterroleidentities
     verbs:
-    - get
-    - list
-    - watch
+      - get
+      - list
+      - watch
+  {{- end}}
+
+  {{ if eq .Values.managementCluster.provider.kind "capz" -}}
   - apiGroups:
-    - aadpodidentity.k8s.io
+      - infrastructure.cluster.x-k8s.io
     resources:
-    - azureidentities
+      - azureclusters
     verbs:
-    - create
+      - get
+      - list
+      - watch
   - apiGroups:
-    - aadpodidentity.k8s.io
+    - infrastructure.cluster.x-k8s.io
     resources:
-    - azureidentitybindings
-    - azureidentitybindings/status
+    - azureclusteridentities
```

Comment:
Are these changes meaning that authentication against Azure will now be done at the cluster level instead of the app one ?

Reply:
No but those are not needed

Reply:
Thoses changes only reduce the crs that this operator needs to access and not just about everything for no good reasons

Reply:
Oh ok nice